unstructured/embed/mixedbreadai.py

```diff
@@ -0,0 +1,217 @@
+import os
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING, List, Optional
+
+import numpy as np
+
+from unstructured.documents.elements import Element
+from unstructured.embed.interfaces import BaseEmbeddingEncoder, EmbeddingConfig
+from unstructured.ingest.error import EmbeddingEncoderConnectionError
+from unstructured.utils import requires_dependencies
+
+USER_AGENT = "@mixedbread-ai/unstructured"
+MAX_BATCH_SIZE = 256
+
+if TYPE_CHECKING:
+    from mixedbread_ai.client import MixedbreadAI
+    from mixedbread_ai.core import RequestOptions
+    from mixedbread_ai.types import EncodingFormat, TruncationStrategy
+
+
+@dataclass
+class MixedbreadAIEmbeddingConfig(EmbeddingConfig):
+    """
+    Configuration class for Mixedbread AI Embedding Encoder.
+
+    Attributes:
```

Comment:
@huangrpablo can you minimize these Attributes to mimic what we are using for the other embedders. And only use what is necessary (api_key, model_name). We can revisit this if a need for the additional arguments comes up. We will have to support Attributes in a UI in our Platform product and it will be very cluttered. 

https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/embed/octoai.py

Reply:
hey, @potter-potter, I removed the unsupported attributes. Could you have a look?